<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formcs</class>
 <widget class="QWidget" name="formcs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>FORMULARIO DE ENROLAMIENTO</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>9</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="formcs_labelImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/User-blue-icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_renewImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Renovar Fotografia</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>461</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_rut"/>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Cargo</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Empresa</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Perfil</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Apellido Paterno:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rut:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="9" column="2">
     <widget class="QComboBox" name="comboBox_profile"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Obtener Informacion</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Apellido Materno</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="lineEdit_position"/>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="lineEdit_Enterprise"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_appPat"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEdit_appMat"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="acceptButton_formcs">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton_formcs">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_manualRegistration">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>62</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enrolamiento Manual</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
